src/sentry/api/serializers/models/organization.py

```diff
@@ -457,6 +458,7 @@ class DetailedOrganizationSerializerResponse(_DetailedOrganizationSerializerResp
     metricsActivateLastForGauges: bool
     extrapolateMetrics: bool
     requiresSso: bool
+    uptimeAutodetection: bool
```

Comment:
since adding this to the response is behind a flag, should this be in `_DetailedOrganizationSerializerResponseOptional` instead?

Reply:
ahha yes it should